src/Portfolio-Rebuild/style.css

```diff
@@ -0,0 +1,217 @@
+  padding: 1%;
+  /* border-radius: 10%; */
+}
+
+/* My Specialities */
+
+.specialities {
+  border: 1px solid grey;
+  background-color: rgb(238, 237, 237);
+  /* display: flex;
+  justify-content: center; */
+  padding: 2%;
+  height: 200px;
+}
+
+/* How I built Skills */
+
+/* inform */
+.contactform{
+  background: -moz-linear-gradient(top, hsl(274, 62%, 95%) 0%, hsl(208, 72%, 89%) 100%);
+  background: -webkit-linear-gradient(top, hsl(194, 87%, 75%) 0%, hsl(208, 89%, 75%) 100%);
+  background: linear-gradient(to bottom, hsl(194, 79%, 74%) 0%, hsl(208, 90%, 69%) 100%);
+  margin-top: 5%;
+  padding: 10%;
+}
+input {
+  border: 2px solid rgb(76, 76, 77);
+  margin-top: -3%;
+}
+
+button {
+  background-color: rgb(0, 102, 255);
+  color: white;
+}
+.myphoto{
+  width: 50%;
+  border-radius: 50%; 
+  margin-bottom: 3%;
+  
+}
+/* button */
+.button{
+  margin: 1%;
+}
+#submit {
+  background: -moz-linear-gradient(top, hsl(194, 100%, 50%) 0%, hsl(208, 100%, 50%) 100%);
+  background: -webkit-linear-gradient(top, hsl(194, 100%, 50%) 0%, hsl(208, 100%, 50%) 100%);
+  background: linear-gradient(to bottom, hsl(194, 100%, 50%) 0%, hsl(208, 100%, 50%) 100%);
+  margin-right: .25rem;
+  padding: 1%;
+}
+
+#submit:hover {
+  background: -moz-linear-gradient(top, hsl(194, 100%, 40%) 0%, hsl(208, 100%, 40%) 100%);
+  background: -webkit-linear-gradient(top, hsl(194, 100%, 40%) 0%, hsl(208, 100%, 40%) 100%);
+  background: linear-gradient(to bottom, hsl(194, 100%, 40%) 0%, hsl(208, 100%, 40%) 100%);
+}
+#reset{
+  background-color: grey;
+  color: white;
+}
+/* footer */
+.footer{
+  margin: 0;
+  padding: 7%;
+  background-color: grey;
+}
+.emifooter{
+  color: white;
+}
+a,.fmenu{
+  display:flex;
+  justify-content: space-around;
+  color: white !important;
+  padding: 3%;
+}
+.icons{
+  display: flex;
+  justify-content: flex-end;
+}
+.icon{
+  width: 5vh;
+}
+@media (max-width: 450px) {
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    width: 95%;
+    height: 200px;
+    margin: 1%;
+    margin-bottom: 650px;
+  }
+  .specialities {
+    height: 200px;
+  }
+  .video {
+    width: 100%;
+    height: 40%;
+  }
+  #message {
+    height: 70px;
+    width: 200px;
+  }
+  .contactform{
+    padding: 3% 3%;
+    text-align: center;
+  }
+  .fmenu{
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+  }
+}
+/* Media for Home Page */
+
+@media (min-width: 451px) and (max-width: 700px) {
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 95%;
+    height: 200px;
+    margin: 1%;
+    margin-bottom: 250px;
+    /* border: 1px solid grey; */
+  }
+  .contactform{
+    display: grid;
+    grid-template-columns: 45% 45%;
+    column-gap: 5%;
+}
+  .video {
+    width: 80%;
+    height: 60%;
+  }
+  #message {
+    height: 130px;
+    width: 250px;
+  }
+  
+}
+@media (min-width: 701px){
+  .grid {
+    display: grid;
```

Comment:
try adding a grid-gap to give space between the boxes. `grid-gap: 1em;`. if you like it, keep it! if not, no worries!

Reply:
Looks much better! Thank you very much!